src/api/offers.js

```diff
@@ -88,6 +90,9 @@ export default ({offerService, errorService}: { offerService: OfferService, erro
       } else if (offer.expirationDate <= Date.now()) {
         const errorResponse = errorService.createOfferExpiredErrorResponse(token)
         response.status(HttpStatus.GONE).json(errorResponse)
+      } else if (offer.confirmed === false && offer.createdDate < Date.now() - CONFIRMATION_PERIOD) {
+        const errorResponse = errorService.createConfirmationLinkExpiredErrorResponse(token)
+        response.status(HttpStatus.GONE).json(errorResponse)
```

Comment:
I think we should hide here the fact that the offer is still in our system. If the cleanup-job hasn't run yet then it returns GONE. As soon as it run it returns NOT_FOUND. I think this is inconsistent.

Reply:
Done